### DISCUSSION TITLE
FXIOS-1814 ⁃ Changing Firefox iOS strings and translations

### DISCUSSION THREAD

**Discussion by imnotabr** [mozilla-mobile/firefox-ios#8502] [General]
I'm interested in doing some translation work for Firefox iOS strings.

Where are the  language string files located in the Firefox iOS source code? I can't find them.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1814)

**Comment by dnarcese** [comment]
Hello, you can help translate our strings at https://pontoon.mozilla.org/projects/firefox-for-ios/! Thanks for your interest.
